ivy/functional/backends/tensorflow/elementwise.py

```diff
@@ -123,15 +123,17 @@ def bitwise_and(
 
 @with_unsupported_dtypes({"2.15.0 and below": ("complex",)}, backend_version)
 def bitwise_invert(
-    x: Union[int, tf.Tensor, tf.Variable],
+    x: Union[int, bool, tf.Tensor, tf.Variable],
     /,
     *,
     out: Optional[Union[tf.Tensor, tf.Variable]] = None,
 ) -> Union[tf.Tensor, tf.Variable]:
-    if "int" not in str(x.dtype):
-        return tf.logical_not(x)
-    else:
-        return tf.bitwise.invert(x)
+    if isinstance(x, bool):
```

Comment:
the [`handle_array_like_without_promotion`](https://github.com/unifyai/ivy/blob/d382882aafb0e19cccc8145cfcaa6cc4f53a77c4/ivy/functional/ivy/elementwise.py#L1153) decorator should be able to convert array-likes and scalars to arrays before the input is passed to the function, is `x` not a `tf.Tensor` inside the backend implementation?

Reply:
No, problem. It's working without the changes. 👍🏻

Reply:
awesome, thanks!